src/main/java/com/redhat/jenkins/plugins/ci/messaging/ActiveMqMessagingWorker.java

```diff
@@ -156,26 +154,15 @@ public boolean connect() {
 
         Connection connectiontmp = null;
         try {
-            connectiontmp = connectionFactory
-                    .createConnection();
+            connectiontmp = connectionFactory.createConnection();
             String url = "";
-            if (Jenkins.getInstance() != null) {
-                url = Jenkins.getInstance().getRootUrl();
+            if (Jenkins.getInstanceOrNull() != null) {
+                url = Jenkins.get().getRootUrl();
             }
-            connectiontmp.setClientID(provider.getName() + "_"
-                    + url + "_" + uuid + "_" + jobname);
+            connectiontmp.setClientID(provider.getName() + "_" + url + "_" + uuid + "_" + jobname);
             connectiontmp.start();
         } catch (JMSException e) {
-            log.severe("Unable to connect to " + provider.getBroker() + " " + e.getMessage());
```

Comment:
do we not want to keep this in case for debugging?

Reply:
Not sure why I removed it. I've added it back.